### ISSUE TITLE
Process finished with exit code -1073740791 (0xC0000409)

### ISSUE THREAD

**Issue by mike-c999** [perrygeo/python-rasterstats#212]
Hi Everybody,
every time I am running a neural net (LSTM in this case) it crashes with the line “Process finished with exit code -1073740791 (0xC0000409)”. It does not happen at a fixed number of epochs. It always depends on the quantity of training/test data I am using to train the net.
Any ideas?
I do not think it is the code but here you go just in case. I am only putting in the net itself.
Thanks so much in advance.

`
# define model
model = Sequential()
model.add(layers.LSTM(units=100,return_sequences=True,input_shape=(batch_size,n_input)))
model.add(layers.LSTM(units=50,return_sequences=True,input_shape=(batch_size,n_input)))
model.add(layers.LSTM(units=25,return_sequences=False,input_shape=(batch_size,n_input)))
model.add(Dense(units=1))

# Create a pic of the model
plot_model(model, to_file='model_plot.png', show_shapes=True, show_layer_names=True)

# compile the model
model.compile(optimizer=optimizers.Adam(learning_rate=0.0001), loss='mse', metrics=['mse'])

# callback: Save the model if it improves
filepath="weights/weights-improvement-{epoch:08d}-{mse:.8f}.hdf5"
checkpoint = ModelCheckpoint(filepath, monitor='mse', verbose=1, save_best_only=True, mode='min')

txtFile = open('mse' + ".txt", "w+", buffering=1)
save_op_callback = LambdaCallback(on_epoch_end = lambda epoch, logs: [createTXT(txtFile,epoch,logs['mse'])],on_train_end = lambda logs: txtFile.close())

# fit the model
callbacks_list = [checkpoint, save_op_callback]
model.fit(X_train, y_train,epochs=999999,batch_size=batch_size,verbose=3,validation_data=(X_test,y_test),callbacks=callbacks_list)
`

**Comment by perrygeo** [comment]
unrealted to rasterstats
